### DISCUSSION TITLE
Recorded Query Management

### DISCUSSION THREAD

**Discussion by ehuggz** [grafana/grafana#48122] [Observability]
The new Recorded Queries feature is proving to be a great, quick fix, for creating historical metrics from SQL Queries for our Client Databases. As it appears necessary to create one query for each of the over 100 databases we currently have in production, the collection of recorded queries is growing quickly. As a result, management of the recorded queries is showing some issues at scale as our collection of business metric queries grows.

Managing 100's of recorded queries becomes a bit daunting given the current UI capabilities.

**Request 1:**  Alphabetize Recorded Query Listing under: Configuration->Recorded Queries
**Request 2:** Add Search bar to filter queries under: Configuration->Recorded Queries
**Request 3:**  It would also be helpful to be able to view the text of the recorded query  to verify the query has been created correctly - if that is possible.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
